src/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -185,7 +187,8 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
                     onChange={setCustomValue}
                     onFocus={() => {
                       onFocus()
-                      jumpTo("price-option-custom")
+                      if (device !== Device.iPhone)
+                        jumpTo("price-option-custom")
```

Comment:
I added the check here instead of inside `useJump` since I assume it works fine in other places (haven't checked)

Should we add it to the [useJump](https://github.com/artsy/force/blob/main/src/Utils/Hooks/useJump.tsx) hook instead?

Reply:
No need to add this to the hook